galaxy_ng/app/migrations/0055_galaxy_role_defs_to_dab_defs.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+
+
+def copy_permissions_to_role_definitions(apps, schema_editor):
+    Permission = apps.get_model('auth', 'Permission')
+    RoleDefinition = apps.get_model('ansible_base.rbac', 'RoleDefinition')
+
+    permissions = Permission.objects.all().filter(name__icontains='namespace')
+    for perm in permissions:
+        RoleDefinition.objects.create(
+            name=perm.name,
+            permissions=[perm.codename],
+            content_type=perm.content_type
+        )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('galaxy', '0054_dab_resource_views'),
```

Comment:
I wonder if the migrations would work if you put in the last migration from the DAB rbac app as a dependency here.